### ISSUE TITLE
Player speed dependant on FPS

### ISSUE THREAD

**Issue by olls** [itsapi/imgui-test#1]
The player moves much faster at 30 FPS than at 120 FPS.

This might be due to us not handling acceleration correctly; I'm not sure we are mixing speed and acceleration terms correctly

**Comment by geraintwhite** [comment]
Does it still happen? I don't notice any difference between 30 and 120.

**Comment by olls** [comment]
Definitely.  It takes ~9 seconds to travel 20 blocks at 30 fps, and ~17 seconds to travel the same at 120 FPS.

I suspect its something to do with the way we are calculating `last_frame_total`.
